src/Installers/Windows/AspNetCoreModule-Setup/CustomAction/aspnetcoreCA.vcxproj

```diff
@@ -49,7 +49,7 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
-    <PlatformToolsetVersion Condition="'$(VisualStudioVersion)' == '17.0' ">v143</PlatformToolsetVersion>
+    <PlatformToolsetVersion Condition="$(VisualStudioVersion) == 17.0 ">v143</PlatformToolsetVersion>
```

Comment:
@rainersigwald is this comparison going to work correctly❔ `$(VisualStudioVersion)` is always defined. My concern is about whether `msbuild` considers `17.0` a "simple alphanumeric value".

Reply:
Hmm, PR validation might work here because we build w/ VS2019 i.e. where the check should be `false`.

Reply:
I don't see the value in removing the single quotes in some places just because we can - it's not obvious at a glance why some conditions need them and some don't (without looking at the documentation). I'd prefer to just keep using single quotes everywhere for consistency

Reply:
But we don't use single quotes everywhere. Our numeric comparisons are inconsistent today. I'm fine removing the last commit i.e. leaving `17.0` alone but would prefer we leave the integer comparisons **un**quoted.

Reply:
In my mind "MSBuild style" is _always single-quote comparands_. The language does allow you to avoid it in some cases but since it requires that you understand whether a property is "always" defined I prefer the safe-but-ugly approach.

Reply:
Guess that makes sense @rainersigwald and @wtgodbe. I'll re-quote everything and will clean up the existing `@({something}->Count()) == 0`, `@({something}->Count()) == '0'`, `@({something}->Count()) == 0`, and `'@({something}->Count())' == '0'` mix in the other direction.

@wtgodbe what's your preference on doing the broader cleanup here versus in a separate PR❔

Reply:
I think it's fine to do that in a separate PR, and it's not super urgent

Reply:
> it's not super urgent

True however I have the regular expressions all set 😄

Reply:
#40108

Reply:
I'm pretty sure we've resolved this but please confirm @wtgodbe

Reply:
Yep, I'm happy